app/cdap/components/RecipeManagement/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IRecipe } from 'components/RecipeList/types';
+
+export interface IRecipeDetailsProps {
```

Comment:
move the interface to its usage file and delete this file

Reply:
as per comment given by @radhikav1 we are maintaining common seperate file for types where we've defined all the types that are being used in Recipe Management, here as we have small interface moved this interface to usage file and removed it in the types file
please suggest if changes needed